src/employee/EmployeeRepository.java

```diff
@@ -64,6 +69,28 @@ public void readParcel() {
         }
     }
 
+    public Employee login (String email, String password){
+        Employee matechedEmployee = employeeList.getEmployeeList().stream().filter(e->e.getEmployeeEmail().equals(email)).collect(Collectors.toList()).get(0);
+
+       if (matechedEmployee != null){
+           if (matechedEmployee.getEmployeePW() == password){
```

Comment:
```suggestion
           if (matechedEmployee.getEmployeePW().equals(password)){
```
String 비교는 equals를 사용해야 안전하게 값을 비교할 것 같아요